Sources/ArcGISToolkit/Documentation.docc/Resources/JobManager/JobManagerViewStep1.swift

```diff
@@ -0,0 +1,48 @@
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct JobManagerTutorialView: View {
+    /// The job manager used by this view.
+    @ObservedObject var jobManager = JobManager.shared
+    /// The job that this view shows data for.
+    @State private var job: Job?
+    /// The job's error in case of failure.
+    @State private var error: Error?
+    /// A Boolean value indicating if we are currently adding an offline map job.
+    @State private var isAddingOfflineMapJob = false
+    
+    var body: some View {
+        VStack {
+            if let job {
+            } else {
+                HStack(spacing: 10) {
+                    if isAddingOfflineMapJob {
+                        ProgressView()
+                    }
+                    Button {
+                        isAddingOfflineMapJob = true
+                        Task {
+                            do {
+                                jobManager.jobs.append(
+                                    try await makeNapervilleOfflineMapJob()
+                                )
```

Comment:
Would it be better to leave this addition until the next step once `makeNapervilleOfflineMapJob()` has been defined? Not sure it matters to much either way.

Reply:
I was trying to make the code additions in one place rather than multiple places. I didn't stick to that everywhere but I felt like that makes it easier to follow. That said, if you think it's better to avoid the compile error you get at the end of that step I can move that code. I could even make it a separate step after the extensions are added to keep it a single block to add.

Reply:
It's probably okay. It gets defined in the very next step.